plugin/src/sbt-test/sbt-mu-srcgen/avroWithSchema/build.sbt

```diff
@@ -1,7 +1,12 @@
 version := sys.props("version")
 
-libraryDependencies ++= Seq(
-  "io.higherkindness" %% "mu-rpc-service" % sys.props("mu")
-)
-
 addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.1" cross CrossVersion.patch)
+
+lazy val root = project
+  .in(file("."))
+  .enablePlugins(SrcGenPlugin)
+  .settings(Seq(
+    libraryDependencies ++= Seq(
+  "io.higherkindness" %% "mu-rpc-service" % sys.props("mu"),
+  )
+  ))
```

Comment:
Could we apply some format here?

Reply:
yeah definitely; I used `sbt scalafmtAll` on these files but it still didn't quite auto-format in the way that I'd hoped.

Reply:
`scalafmtAll` is a bit misleading (that's my fault, I named it!). It doesn't include the sbt files.

`sbt scalafmtSbt` will format the project's sbt files. But this file is actually test data, not part of the project's sbt files, so I'm afraid you'll have to format this one manually.

Reply:
hah that's not a problem and thanks for clarifying!  Should've double-checked the files after running the command.